tabular/src/autogluon/tabular/task/tabular_prediction/dataset.py

```diff
@@ -18,48 +17,29 @@ class TabularDataset(pd.DataFrame):
-        Path to the data file (may be on local filesystem or URL to cloud s3 bucket). 
-        At least one of `file_path` and `df` arguments must be specified when constructing a new `TabularDataset`.
-    df : `pandas.DataFrame` (optional)
-        If you already have your data in a pandas Dataframe, you can directly provide it by specifying `df`. 
-        At least one of `file_path` and `df` arguments must be specified when constructing new `TabularDataset`.
-    feature_types : dict (optional)
-        Mapping from column_names to string describing data type of each column. 
-        If not specified, AutoGluon's fit() will automatically infer what type of data each feature contains.
-    subsample : int (optional)
-        If specified = k, we only keep first k rows of the provided dataset.
-    name : str (optional)
-        Optional name to assign to dataset (has no effect beyond being accessible via `TabularDataset.name`).
-    copy : bool (optional, default=False)
-        If True and `df` is passed, then `df` will be deep copied, resulting in two instances of the provided data in memory.
-        If False, the original `df` and the new `TabularDataset` will share the same underlying data.
-        Set `copy=True` if you don't want AutoGluon to be able to modify the original DataFrame during it's training process (and your machine has enough memory).
+    data : :class:`pd.DataFrame` or str
+        Path to data file (csv or parquet) or loaded :class:`pd.DataFrame`.
+        If you already have your data in a :class:`pd.DataFrame`, you can directly provide it by specifying `data`.
 
     Attributes
     ----------
-    name: (str)
-        An optional name assigned to this `TabularDataset`.
     file_path: (str)
         Path to data file from which this `TabularDataset` was created.
-    feature_types: (dict) 
-        Maps column-names to string describing the data type of each column in this `TabularDataset`.
-    subsample: (int) 
-        Describes size of subsample retained in this `TabularDataset` (None if this is original dataset).
+        None if `data` was a :class:`pd.DataFrame`.
     
     Note: In addition to these attributes, `TabularDataset` also shares all the same attributes and methods of a pandas Dataframe. 
-    For detailed list, see:  https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html
+    For a detailed list, see:  https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html
 
     Examples
     --------
     >>> from autogluon.tabular import TabularDataset
-    >>> train_data = TabularDataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
-    >>> test_data = TabularDataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
+    >>> train_data = TabularDataset('https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
+    >>> test_data = TabularDataset('https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
```

Comment:
can probably omit test_data from this example

Reply:
omitted